nodestream/databases/neo4j/query_executor.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         chunk_size: int = 1000,
         execute_chunks_in_parallel: bool = True,
         retries_per_chunk: int = 3,
+        alias: str = "neo4j-default",
```

Comment:
Why introduce another value here? Why not rely on the URI and log it out?  See [here](https://github.com/nodestream-proj/nodestream/blob/280298babb589743aa544074a3bd6a45cdf44001/nodestream/databases/neo4j/query_executor.py#L94).

Reply:
Figured we wouldn't want the uri in the logs, but I can add it into the log message instead.